### ISSUE TITLE
Kevo Lock app

### ISSUE THREAD

**Issue by samuelclay** [samuelclay/turntouch-ios#7]
Here's a Python api to crib from: https://github.com/cseelye/pykevoplus/blob/master/pykevoplus/__init__.py. It just uses a web api with a username and password (which we can ask for in the app in an options view on the app).

**Comment by Dejal** [comment]
I just tried to sign in to my Kevo lock using the logic gleaned from that script, but it appears to be broken by the captcha; apparently a recent addition. I can't get it to log in. 

If you have a Kevo and can try the script directly, let me know if you can get it to work.

**Comment by samuelclay** [comment]
I unfortunately don't have a Kevo. It's likely it's broken on purpose. Nest has just gone down this route.

**Comment by Dejal** [comment]
Yeah, seems like. I guess this idea should be either closed, or changed to low priority or similar.
